examples/performance/00_overview.ipynb

```diff
@@ -340,9 +340,9 @@
       "    {\n",
       "      for (int y0_blk0 = y_m; y0_blk0 <= y_M; y0_blk0 += y0_blk0_size)\n",
       "      {\n",
-      "        for (int x = x0_blk0; x <= MIN(x0_blk0 + x0_blk0_size - 1, x_M); x += 1)\n",
+      "        for (int x = x0_blk0; x <= MIN(x_M, x0_blk0 + x0_blk0_size - 1); x += 1)\n",
```

Comment:
It's symbolic sorting, it's deterministic it just sorts the arguments based on sympy rules (string sort, numbers, ...). I think it's a really bad idea to enforce our own sorting, it's a lot of extra work that needs to be tested and maintain rather than using an existing and pretty robust symbolic sorting.